runtime/inspector/mod.rs

```diff
@@ -786,46 +402,41 @@ impl task::ArcWake for InspectorWaker {
   }
 }
 
-struct DenoInspectorSession {
+/// An inspector session that proxies messages to Websocket connection.
+struct WebsocketSession {
   v8_channel: v8::inspector::ChannelBase,
-  v8_session: v8::UniqueRef<v8::inspector::V8InspectorSession>,
```

Comment:
So originally this was ` v8::UniqueRef` and I changed it to `v8::UniquePtr` to follow what's used in `DenoInspector` - to be honest I don't know what's the difference in usage of those - we get `UniqueRef` from v8 but we're changing it to `UniquePtr` using `.into()` call. I guess we could just use `UniqueRef` instead? 

Note that it's been wrapped in `Rc<RefCell<>>` so it's clonable and can be passed to other structures require it instead of passing raw pointers.

Reply:
`UniqueRef<T>` cannot be null, so it's like `&mut T`.
`UniquePtr<T>` can be null, so it's like `Option<&mut T>`.

If V8 gives you an `UniqueRef` then you can keep it that way.
However if feel the need to use `Option<UniqueRef<T>>`, then `UniquePtr<T>` is preferrable.